lib/client/directives/widgetDirective.js

```diff
@@ -12,7 +12,9 @@ module.exports = function($templateRequest, $compile, $interpolate) {
 
             var templates = [
                 '/templates/' + $attrs.type + '.html',
-                '/templates/' + $attrs.type + '-' + $attrs.id + '.html'
+                '/templates/' + $attrs.type + '-' + $attrs.id + '.html',
+                '/templates/' + $attrs.board + '/' + $attrs.type + '.html',
+                '/templates/' + $attrs.board + '/' + $attrs.type + '-' + $attrs.id + '.html'
```

Comment:
please use dashes to separate board name from type as we also do with type and id. For example `testboard-clock-10.html`